Examples/ExamplesApp/AnyExample.swift

```diff
@@ -16,9 +16,9 @@ import SwiftUI
 
 struct AnyExample<Content: View> {
     var name: String
-    var content: @MainActor () -> Content
+    var content: () -> Content
     
-    init(_ name: String, content: @autoclosure @escaping @MainActor () -> Content) {
+    init(_ name: String, content: @autoclosure @escaping () -> Content) {
```

Comment:
What's the reason for these changes?

Reply:
Leaving these annotations in place produces the following errors in `Examples.swift`.

<img width="920" alt="Screenshot 2024-10-02 at 11 01 38 AM" src="https://github.com/user-attachments/assets/b711884e-656a-43cf-845a-47f596a0500d">

Given that `View`s are now annotated `@MainActor` in Xcode 16, I don't understand why some of these initializers are erroring while others are not. Despite this, these changes match `AnyExample` in the Swift Examples app.

Reply:
I see. I didn't see the errors, but that's because I was building the toolkit itself instead of the Examples app.